modules/age.nix

```
@@ -201,6 +212,41 @@ in
         "agenixChownKeys"
       ];
     };
+
+    # services that watch for file changes and exectue the configured action
+    systemd.services = lib.mkMerge
+      (lib.mapAttrsToList
+        (name: {action, service, file, path, mode, owner, group, ...}:
+          let
+            fileHash = builtins.hashString "sha256" (builtins.readFile file);
```

age files aren't guaranteed to be valid Nix strings (`example/secret1.age` isn't, for example)

```suggestion
            fileHash = builtins.hashFile "sha256" file;
```

---

Oh, good call! that's cleaner anyways.